Out of source build with proto is broken

**How to reproduce:**
1. `./autogen.sh`
2. `mkdir /tmp/pibuild`
3. `cd /tmp/pibuild`
4. `/path/to/PI/configure --with-proto`
5.  `make`

**Error log:**
```
libtool: compile:  g++ -DHAVE_CONFIG_H -I. -I/.autodirect/sysgwork/guysz/PI/proto/server -I.. -I/.autodirect/sysgwork/guysz/PI/proto/frontend -isystem ../cpp_out -isystem ../grpc_out -Wall -Werror -g -O2 -MT pi_server.lo -MD -MP -MF .deps/pi_server.Tpo -c /.autodirect/sysgwork/guysz/PI/proto/server/pi_server.cpp  -fPIC -DPIC -o .libs/pi_server.o
/.autodirect/sysgwork/guysz/PI/proto/server/pi_server.cpp:40:10: fatal error: server_config/server_config.h: No such file or directory
   40 | #include "server_config/server_config.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[4]: *** [Makefile:518: pi_server.lo] Error 1
make[4]: Leaving directory '/tmp/pibuild/proto/server'
make[3]: *** [Makefile:1693: all-recursive] Error 1
make[3]: Leaving directory '/tmp/pibuild/proto'
make[2]: *** [Makefile:733: all] Error 2
make[2]: Leaving directory '/tmp/pibuild/proto'
make[1]: *** [Makefile:452: all-recursive] Error 1
make[1]: Leaving directory '/tmp/pibuild'
make: *** [Makefile:384: all] Error 2
```

--- a/proto/server/Makefile.am
+++ b/proto/server/Makefile.am
@@ -1,6 +1,7 @@
 ACLOCAL_AMFLAGS = ${ACLOCAL_FLAGS} -I m4
 
 AM_CPPFLAGS = \
+-I$(top_srcdir) \
 -I$(top_srcdir)/frontend \
 -isystem $(top_builddir)/cpp_out \
 -isystem $(top_builddir)/grpc_out

